src/utils/read-swagger.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { SwaggerDefinition } from '../types';
+
+export function readSwagger(dir: string): SwaggerDefinition {
+    const fileNames = fs.readdirSync(dir);
+
+    if (fileNames.length === 0) {
+        throw new Error(`Swagger directory ${dir} is empty.`);
+    }
+
+    if (fileNames.length !== 1) {
+        throw new Error(`Too many swagger definition files.`);
```

Comment:
We could make this error a bit more clear to what's wrong, like "More than one Swagger definition file found in Swagger directory"